tests/test_ecommerce.py

```diff
@@ -465,19 +465,29 @@ def test_metadata():
                     "enum": ["httpResponseBody", "browserHtml"],
                 },
                 "crawl_strategy": {
-                    "default": "full",
+                    "default": "automatic",
                     "description": "Determines how the start URL and follow-up URLs are crawled.",
                     "enumMeta": {
+                        "automatic": {
+                            "description": (
+                                "Follow pagination, subcategories, and product detail pages. "
+                                "If starting on a homepage, it would attempt to discover other "
+                                "URLs in the page using heuristics."
```

Comment:
Do you think we'll extend the heuristics in the future, while keeping the name?

If so, it could make sense to document this strategy as "automatic". We may still explain the exact behavior, as an implementation detail which can change.

Reply:
Good call. Yeah, we'll definitely have some changes soon like some good utilization of sitemaps for better item coverage.

What do you think of this new one? https://github.com/zytedata/zyte-spider-templates/pull/52/commits/ad2776ac41f2bd8f0cf871ff6450f1e5e0c8cfe2